### ISSUE TITLE
Fix flux for input/data/2014/2014ApJ...788..158A/tev-000055.yaml

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-cat#184]
@GernotMaier - could you please have a look at the flux in `input/data/2014/2014ApJ...788..158A/tev-000055.yaml`?

Currently you have this:
https://github.com/gammapy/gamma-cat/pull/181/files#diff-cf7e0e5e5bb3e5b62f36575ba084f49fR14
```
 flux: {val: 2.6, err: 0.7, scale: 1e-12, unit: erg cm-2 s-1}
```
This doesn't work with our schema, we can't have an energy flux in the `flux` field, it has to be a photon flux.

I the paper http://iopscience.iop.org/article/10.1088/0004-637X/788/2/158/pdf I see

> The 1 TeV energy flux is (2.6 ± 0.7)×10−12 erg cm−2 s−1 and the photon index is Γ = 3.15 ± 0.23.

I think this is an integral energy flux above 1 TeV?
(or is it the differential energy flux `e2dnde` at 1 TeV?)

If this is correct, then we need to convert this to the equivalent photon flux, either by doing it manually on data entry, or by extending our schema and scripting to allow `eflux` for `pl2`, and if present, convert to `flux` in our scripts in gamma-cat.
I think in any case computing the conversion factor is non-trivial, and one has to take the approach similar to  https://github.com/gammapy/gamma-cat/issues/74 .

**Comment by GernotMaier** [comment]
Used the paper documentation to get the right normalisation value, see pull request https://github.com/gammapy/gamma-cat/pull/192

**Comment by cdeil** [comment]
This issue was resolved by @GernotMaier recently in #192 and I just forgot to close it.
See change here: https://github.com/gammapy/gamma-cat/pull/192/files#diff-cf7e0e5e5bb3e5b62f36575ba084f49fR14
